src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanUpFixer.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Language.CodeCleanUp;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup
+{
+    internal abstract class CodeCleanUpFixer : ICodeCleanUpFixer
```

Comment:
➡️ We need to be able to import instance of `ICodeCleanUpFixer` which are relevant for Roslyn. Since `ICodeCleanUpFixer` can apply to other languages/extensions, we use this abstract class as the base type for Roslyn features and MEF exports.